dashboards-observability/public/components/integrations/sdk/templates/indexTemplate.js

```diff
@@ -0,0 +1,410 @@
+      },
+      {
+        name: 'Metrics',
+        icon: 'Metrics',
+        path: 'create?type=integration',
+        description: 'Monitor connection and request metrics with Kibana',
+      },
+      {
+        name: 'Apache',
+        icon: 'Apache',
+        path: 'create?type=integration&app=Apache',
+        description: 'Monitor connection and request metrics with Apache',
+      }
+    ]`,
+    apacheconstant : `export enum ApacheConfig {
+      STEP_1 = 'Step 1',
+      STEP_2 = 'Step 2',
+      STEP_3 = 'Step 3',
+      STEP_4 = 'Step 4',
+      STEP1_DESC = 'In this step we have added the custom log that we want to retrieve in the log.',
+      STEP2_DESC = 'In this step we need to provide the location of the access file.',
+      STEP3_DESC = 'In this step we need to update the fluent configuration file (fluent.conf)',
+      STEP4_DESC = 'In this step we need to provide the location of the access file.',
+      STEP5_DESC = 'In this setting we have created nginx_json_format as index to fetch data.',
+      CONFIGURATION_STEP1 = \`LogFormat " {"timestamp":"%{%d/%b/%Y %T}t.%{msec_frac}t %{%z}t", "process_time_taken_microseconds":"%D", "filename":"%f", "remoteIP":"%a", "host":"%V", "request_url":"%U", "query":"%q", "request_method":"%m", "redirected_request_status":"%>s", "userAgent":"%{User-agent}i", "referer":"%{Referer}i", "localIP":"%A", "cookie_varname":"%{VARNAME}C", "content_of_varname":"%{VARNAME}e", "remote_host_name":"%h", "request_protocol":"%H", "keepalive_request":"%k", "remote_logname":"%l", "canonical_port":"%p", "query_string":"%q", "first_line_request":"%r", "time_to_serve_Request":"%T", "authenticated_user":"%u", "canonical_servername":"%v", "connection_response_status":"%X", "byte_received":"%I", "byte_sent":"%O", "pid_child":"%P", "thread_id":"%{format}P", "status":"%s" }" custom\`,
+      CONFIGURATION_STEP2 = \`##this uses the our custom log format access_log /var/log/nginx/json_format.log custom_format\`,
+      CONFIGURATION_STEP3 = \`<source>
+      @type tail
+       <parse>
+         @type json
+         types bytes_sent:integer,byte_received:integer
+         #time_key time
+         #time_format %time
+         #time_format %d/%b/%Y:%H:%M:%S %z
+        </parse>
+       path /var/log/apache2/custom.log
+       pos_file /home/kbhise/apache_pos2
+       tag  apache2
+      </source>\`,
+      CONFIGURATION_STEP4 = \`<match apache2>
+      @type opensearch
+      ssl_verify false
+      host localhost
+      port 9200
+      index_name apache_log_final
+      verify_os_version_at_startup false
+      suppress_type_name true
+      #include_timestamp true
+     </match>\`,
+      CONFIGURATION_STEP5 = \`"_source" : {
+        "@timestamp" : "2022-06-13T09:56:19.609Z",
+        "timestamp" : "13/Jun/2022 15:26:18.481 +0530",
+        "process_time_taken_microseconds" : "1128427",
+        "filename" : "/var/www/wordpress/index.php",
+        "remoteIP" : "172.24.176.1",
```

Comment:
Do not use actual IP addresses. Use something like 127.0.0.1 or 1.2.3.4 - something that looks like dummy data in the template file.